assets/agenda/components/AgendaListItemIcons.tsx

```diff
@@ -94,7 +94,7 @@ class AgendaListItemIcons extends React.Component<IProps, IState> {
 
     render() {
         const props = this.props;
-        const state = this.state;
+        const state = this.getUpdatedState();
```

Comment:
I wonder why the state is not updated when the props changed, we should rather fix that imo